kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/ErrorHandlingExecutorCallAdapterFactory.java

```diff
@@ -153,7 +153,9 @@ public Response<T> execute() {
         }
       } catch (JsonProcessingException jpe) {
         throw new SpinnakerConversionException(
-            "Failed to process response body", jpe, delegate.request());
+            "Failed to process response body: " + jpe.getOriginalMessage(),
```

Comment:
What do you think of:
```suggestion
            "Failed to process response body: " + jpe.getMessage(),
```
With getOriginalMessage, the resulting message is e.g.:
```
Failed to process response body: Unexpected end-of-input: was expecting closing quote for a string value
```
With getMessage, the resulting message:
```
Failed to process response body: Unexpected end-of-input: was expecting closing quote for a string value
 at [Source: (okhttp3.ResponseBody$BomAwareReader); line: 1, column: 29]
```
I think the extra detail is helpful.